bpf/custom/bpf_custom.c

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright (C) 2020 Authors of Cilium */
+
+#include <bpf/ctx/skb.h>
+#include <bpf/api.h>
+
+#include "lib/common.h"
+
+#define DIRECTION_INGRESS	0
+#define DIRECTION_EGRESS	1
```

Comment:
nit: Do we need these defined here as well as `CUSTOM_CALLS_IDX_INGRESS`, `CUSTOM_CALLS_IDX_EGRESS` since the caller is using the latter to pass through? Or if we think it's nicer to read, maybe we should define these against the aforementioned constants?